Formula/container-diff.rb

```diff
@@ -0,0 +1,254 @@
+class ContainerDiff < Formula
+  desc "Diff your Docker containers"
+  homepage "https://github.com/GoogleCloudPlatform/container-diff"
+  url "https://github.com/GoogleCloudPlatform/container-diff/archive/v0.5.2.tar.gz"
+  sha256 "9f64267b01e9f96efd053eadbac6435bdb15c05597e8ebe4e7081e13fb7e45cf"
+  head "https://github.com/GoogleCloudPlatform/container-diff.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkpath "src/github.com/GoogleCloudPlatform"
```

Comment:
`(buildpath/"src/github.com/GoogleCloudPlatform").mkpath`

Reply:
🙇  Added in 81f3661cbd. ♻️